plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -281,13 +378,33 @@ export const resolvers = (): IResolvers => ({
     ): Observable<JobConnection | null> {
       return Observable.of(null);
     },
-    job(_obj = {}, _args: GeneralArgs, _context = {}): Observable<Job | null> {
-      return Observable.of(null);
+    job(_obj = {}, args: GeneralArgs, _context = {}): Observable<Job | null> {
```

Comment:
Nice. I think I can see the direction here now. 

I outlined down here an approach that I think could achieve the same goals and could have saved us a couple days of work. I do that because I still think we can save time for the jobs resolver if we do that (see comment bellow).

```js
return responses$
  .map(response => new Job(response))
  .map(job => toGraphqlType(job))
```

```js
function toGraphqlJob(job) {
   return {
         id: job.getId(),
         description: job.description,
         ///....
         activeRuns: job.getActiveRuns()
      };
}
```

Reply:
For the jobs resolver it would be something like:

```
return responses$
  .map(responseList => responseList.map((jobData) => new Job(jobData))
  .map(jobs => new JobList({ items: jobs }))
  .map(jobsList => jobsList.filter(...).sort(...).whatever(..))
  .map(finalJobsList => jobstList.getItems())
  .map(jobs => jobs.map((job) => toGraphqlType(job))
```

Reply:
I dont wanna revive Friday discussion. I just wanted to provide that piece of example of how I think we could solve it simple. On the case we don't see equal, I am commited to drive what you have already decided.

Reply:
@orlandohohmeier, @juliangieseke @DanielMSchmidt